iam.yml

```diff
@@ -57,6 +57,12 @@ Resources:
                   - "execute-api:ManageConnections"
                 Resource:
                   - !Sub "arn:aws:execute-api:*:${AWS::AccountId}:${JavabuilderApiId}/*"
+              # Build and Run Lambdas need to be able to write metrics to CloudWatch
+              - Effect: Allow
+                Action:
+                  - 'cloudwatch:PutMetricData'
+                Resource:
+                  - '*'
```

Comment:
Not critical, but could we scope this to `javabuilder*` and then use that namespace for any metric that Javabuilder publishes?

Reply:
If I'm reading [the documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/permissions-reference-cw.html) right, it seems like only `*` is supported here